plugins/BEdita/API/src/Controller/TrashController.php

```diff
@@ -32,6 +33,30 @@ class TrashController extends AppController
      */
     public $modelClass = 'Objects';
```

Comment:
Is `$this->Objects` still used? If not, we might remove `$modelClass` and `@property` annotation in class' docblock.

Reply:
It is still used in TrashController::index() listing all trashed objects.